README.md

```diff
@@ -1,18 +1,24 @@
 PGObserver
 ==========
 
-PGObserver was developed at Zalando during the past years to monitor performance metrics of our different PostgreSQL clusters. Due to our use of a stored procedure API layer our strong focus was on monitoring that API layer. However the number of metrics was increased to include other relevant values:
+PGObserver is a monitoring solution for PostgreSQL databases, covering almost all metrics provided by the database engine's internal statistics collector.
+PGObserver works out of the box on all Postgres databases from version 9.0 (including AWS RDS) and does not require installing any server side database extensions nor privileged users, making it perfect for developers!
```

Comment:
... version 9.0 (including AWS RDS) ..
sound like RDS has something to do with versions ..
better would be
on all Postgres databases (including AWS RDS) from version 9.0 ...